guides/common/modules/proc_upgrading-the-external-database-operating-system.adoc

```diff
@@ -0,0 +1,30 @@
+[id="upgrading-the-external-database-operating-system"]
+= Upgrading the external database operating system
+
+Optional: You can upgrade the external database from {EL} 8 to {EL} 9 using backup and restore.
```

Comment:
`Optional:` is AFAIK typically used in procedure steps. In introductions, I most often see it written as a regular part of the sentence, like this:

```suggestion
Optionally, you can upgrade the external database from {EL} 8 to {EL} 9 using backup and restore.
```

The reason is that an introduction is just regular text so it should follow a normal sentence flow.